sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -552,6 +552,14 @@ public synchronized void onActivityPaused(Activity activity) {
         }
     }
 
+    @Override
+    public void onApplicationEnterForeground() {
+        if (mChannel != null) {
```

Comment:
Do we need to keep the former activity restart check in onActivityCreated? I imagine this is redundant now that we have this new callback.

Basically, try removing onActivityCreated and see if this scenario is still working: click back button to close app (not home), immediately launch app from app menu: does a new check happen? If yes we can remove the old onActivityCreated callback code.

Reply:
We checked. It works expected.